<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Plot</class>
 <widget class="QDialog" name="Plot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1196</width>
    <height>680</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1196</width>
    <height>680</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1196</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GRAPH</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #4abdac;
</string>
  </property>
  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>731</width>
     <height>631</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>731</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;</string>
   </property>
  </widget>
  <widget class="QLabel" name="show">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>30</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font: 700 18pt &quot;Orbitron&quot;;
background-color: #4abdac;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="show_2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>70</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font: 700 18pt &quot;Orbitron&quot;;
background-color: #4abdac;</string>
   </property>
   <property name="text">
    <string>x min: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="show_3">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>140</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font: 700 18pt &quot;Orbitron&quot;;
background-color: #4abdac;</string>
   </property>
   <property name="text">
    <string>x max: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="show_4">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>210</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font: 700 18pt &quot;Orbitron&quot;;
background-color: #4abdac;</string>
   </property>
   <property name="text">
    <string>y min: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="show_5">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>280</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font: 700 18pt &quot;Orbitron&quot;;
background-color: #4abdac;</string>
   </property>
   <property name="text">
    <string>y max: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="buttonXMin">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>70</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>55555</width>
     <height>555555</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
   width: 100%;
    height: 80px;
    border: 3px solid black;
    font-size: 2rem;
    color: black;
font: 700 18pt &quot;Orbitron&quot;;;
    background: white;
    
}
QDoubleSpinBox:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>-10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="buttonYMin">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>210</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>55555</width>
     <height>555555</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
   width: 100%;
    height: 80px;
    border: 3px solid black;
    font-size: 2rem;
    color: black;
font: 700 18pt &quot;Orbitron&quot;;;
    background: white;
    
}
QDoubleSpinBox:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>-10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="buttonXMax">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>140</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>55555</width>
     <height>555555</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
   width: 100%;
    height: 80px;
    border: 3px solid black;
    font-size: 2rem;
    color: black;
font: 700 18pt &quot;Orbitron&quot;;;
    background: white;
    
}
QDoubleSpinBox:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="buttonYMax">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>280</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>55555</width>
     <height>555555</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
   width: 100%;
    height: 80px;
    border: 3px solid black;
    font-size: 2rem;
    color: black;
font: 700 18pt &quot;Orbitron&quot;;;
    background: white;
    
}
QDoubleSpinBox:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
   <property name="value">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_graph">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>450</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   width: 100%;
    height: 80px;
    border-radius: 20px;
    border: 3px solid black;
    font-size: 2rem;
    color: black;
    font-family: 'Orbitron', serif;
background: #e17055;
    
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="text">
    <string>Build Graph</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="show_6">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>350</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font: 700 18pt &quot;Orbitron&quot;;
background-color: #4abdac;</string>
   </property>
   <property name="text">
    <string>Step: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="step">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>350</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>55555</width>
     <height>555555</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
   width: 100%;
    height: 80px;
    border: 3px solid black;
    font-size: 2rem;
    color: black;
font: 700 18pt &quot;Orbitron&quot;;
    background: white;
    
}
QDoubleSpinBox:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
   <property name="value">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="error">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>580</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 100%;
    height: 80px;
    border: 3px solid black;
    font-size: 2rem;
    color: black;
font: 700 20pt &quot;Orbitron&quot;;
    background: white;</string>
   </property>
   <property name="text">
    <string>NO ERRORS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>PlotLibrary/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
